lib/views/help/requesting.es.html.erb

```diff
@@ -383,10 +383,7 @@
         </li>
       </ol>
       <p>
-        En estos casos puedes utilizar los recursos legales planteados en el punto 19.
-      </p>
-      <p>
-        Tendrás la posibilidad de categorizar tu respuesta según una lista de categorías (ver punto 17). Si no sabes cómo hacerlo te ayudaremos. <i>derechoapreguntar.org</i>
+        En estos casos puedes utilizar los recursos legales planteados en esta web: <a href="#recursos">Los Recursos en Nicaragua</a>
```

Comment:
Ah, I can see you've added a custom anchor for Nicaragua below. Not sure why this link isn't taking me to it – will have a look in to it.